src/store/search.js

```diff
@@ -0,0 +1,52 @@
+import Vue from 'vue'
+import clone from 'clone'
+import { debounce } from '../utilities/debounce'
+import { searchIndex } from '../utilities/lunr'
+
+export const filter = {
+  NONE: { name: 'All results', value: 'NONE' },
+  API_DOCS: { name: 'API Docs', value: 'API_DOCS' },
+  DEV_CENTER: { name: 'Documentation', value: 'DEV_CENTER' },
```

Comment:
Can we name this `Developer Docs`, same as the section in the side nav?

Reply:
Good call! Updating.